### DISCUSSION TITLE
Show only new Version

### DISCUSSION THREAD

**Discussion by ghost** [red6/pdfcompare#105] [Q&A]
Hello,
I am just trying out PdfCompare and was wondering, if it is possible to only show the "new" text, not the old one. It is impossible to read when the text is overlapping, no matter which color.
When I tried setting the actual color to null, it still used the default red color to show the differences.
Any idea on how to only show the updated version?
Thank you in advance!

**Comment by finsterwalder** [comment]
You can configure the color of expected and actual pixels.
See the README how to configure this.
Defaults are red for actual and green for expected, to clearly show what was expected and what was actually rendered.
When you change the expected color to something like gray or black, it's probably not as distracting.
There is no option to not show expected differences currently.

**Reply by ghost** [reply]
Thank you for your fast reply.
I was afraid so, my client wants to use it like an compare version in Word, where you see the differences of the document, just with generated PDF reports. Too bad, this seemed like an super easy way - but thanks again for your answer, so I know I don't need to search anymore :)
